HACKING.md

```diff
@@ -201,6 +201,27 @@ EOF
 oc create secret generic aws-fcos-kola-bot-config --from-file=config=/path/to/kola-secret
 ```
 
+### [OPTIONAL] Creating GCP credentials configs
+
+If you are in production where we upload images to GCP OR you want to
+test uploading to GCP as part of your pipeline development, you need to
+create a credentials config as a secret within OpenShift.
+
+First grab a json file that represents the credentials for your service account
+from GCP. Then create the secret in OpenShift:
```

Comment:
Can we put more details here on where to get the JSON file from/what format it is?

Reply:
fixed